cluster/charts/rook-ceph/templates/deployment.yaml

```
@@ -160,6 +160,10 @@ spec:
           value: {{ .Values.csi.resizer.image | quote }}
 {{- end }}
 {{- end }}
+{{- if .Values.csi.podLabels }}
+        - name: ROOK_CSI_POD_LABELS
+          value: "{{ .Values.csi.podLabels }}"}}
```

Looks like there are some extra brackets, and also we've been using the `| quote` more often than putting actual quotes around the value, although we certainly have a mix of both approaches.
```suggestion
          value: {{ .Values.csi.podLabels | quote }}
```